index.js

```diff
@@ -598,7 +598,7 @@ class GithubScm extends Scm {
         const command = [];
 
         command.push(
-            "export SD_GIT_WRAPPER=\"$(if [ `uname` = 'Darwin' ]; " +
+            "export SD_GIT_WRAPPER=\"$(if [[ `uname` = 'Darwin' || $SD_HAB_ENABLED ]]; " +
```

Comment:
Is this inverse? it looks using `eval` when `SD_HAB_ENABLED` is true.

Reply:
@tk3fftk Thaks, fixed it in this https://github.com/screwdriver-cd/scm-github/pull/199